### PR TITLE
Style guide tweaks

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/views/Interstitial.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import React from 'react';
 
 export const Interstitial = () => (
-  <div className="progress-indicator">
+  <div className="progress-indicator" aria-live="assertive">
```
Comment (kkoskelin) [inline web-client/src/views/Interstitial.jsx]: OK, but without a message (e.g. aria-label="one moment") the assertive value may not be entirely useful.

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Try providing an aria-label="your description" and I think we'll be good.

File: web-client/src/views/Interstitial.jsx (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/views/Interstitial.jsx]: Think we should just remove it?

Comment (ericsorenson) [reply web-client/src/views/Interstitial.jsx]: @kkoskelin updated!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**